pages/apis/graphql/graphql_cookbook.md

```diff
@@ -906,3 +906,25 @@ mutation UpdateTeamPipelineReadonly {
   }
 }
 ```
+
+## Test Analytics
+
+### Get number of builds between 2 dates 
```

Comment:
Style nitpicks: 

- Write out number for 0-9.
- Add a blank line between headings and paragraphs.

```suggestion
### Get number of builds between two dates
 
```